internal/internal_task_handlers.go

```diff
@@ -424,7 +426,7 @@ func (w *workflowExecutionContextImpl) onEviction() {
 	w.mutex.Unlock()
 }
 
-func (w *workflowExecutionContextImpl) isDestroyed() bool {
+func (w *workflowExecutionContextImpl) IsDestroyed() bool {
```

Comment:
Is it safe to check without locking?

Reply:
No. But they are all called with lock obtained.